data/src/main/java/com/anytypeio/anytype/data/auth/status/P2PStatusRemoteChannel.kt

```diff
@@ -6,14 +6,14 @@ import com.anytypeio.anytype.domain.workspace.P2PStatusChannel
 import kotlinx.coroutines.flow.Flow
 
 interface P2PStatusRemoteChannel {
-    fun observe(activeSpaceId: Id): Flow<List<P2PStatusUpdate>>
+    fun observe(activeSpaceId: Id): Flow<P2PStatusUpdate>
```

Comment:
Why not call it `P2PStatus`? "update" is redundant, because status basically means "update"

Reply:
There's already a P2PStatus model. But we can use P2PStatusEvent instead P2PStatusUpdate